test/unit/uk/gov/hmrc/individualincomedesstub/controller/EmploymentControllerSpec.scala

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.individualincomedesstub.controller
+
+import org.joda.time.LocalDate._
+import org.mockito.Mockito._
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.mock.MockitoSugar
```

Comment:
This is deprecated in `scalatest` version `3.0.1` and hence throws up compile time warnings. We should use the new `org.scalatest.mockito.MockitoSugar` instead.